README.md

```diff
@@ -29,11 +29,24 @@ There is also an experimental binary which uses Tessera's POSIX storage backend.
 
 TesseraCT is under active development, and will reach alpha in 2025Q3 🚀.
 
-At the moment, TesseraCT supports Amazon Web Service and Google Cloud Platform.
-Read the FAQ to understand [why we chose these platforms](#why-these-platforms),
-or if [you're interested in others](#can-i-run-tesseract-outside-gcp-or-aws).
-
-[Public test instances](#test_tube-public-test-instances) run on GCP.
+|Platform         |Architecture            |Our use-case          |Performance|Binary                                  |Deployment                                                      |
+|-----------------|------------------------|----------------------|-----------|----------------------------------------|----------------------------------------------------------------|
+|GCP              |Spanner + GCS + MIG     |public staging logs   |TODO       |[gcp](/cmd/tesseract/gcp/main.go)       |[doc](/deployment/live/gcp/static-ct-staging/logs/arche2025h1/) |
```

Comment:
This is the first time `MIG` appears in the README. I would suggest adding a link to explain what it is.

https://cloud.google.com/compute/docs/instance-groups#managed_instance_groups

Reply:
I linked everything